### ISSUE TITLE
Host not updated and secret and other important files missing.

### ISSUE THREAD

**Issue by rippleitinnz** [EvernodeXRPL/evernode-host#118]
I have been running Evernode for many months without issue.

Today I noted that my instance was offline and had been for 2 days.

It seems two days ago something happened, that caused my instance to self destruct.

The problem:  

root@rippleitinnz:/home# evernode status
evernode: command not found

Evernode does not exist.
My secret has been wiped, but my .cfg remains.
The last version I had (from the dashboard) is v 0.7.1 - it appears 0.7.2 is current.
So, why didn't it autoupdate?
The bigger question is why did my secret get wiped? The secret is empty, so who or how could my secret get destroyed?
root@rippleitinnz:/etc/sashimono/mb-xrpl# dir
governance.cfg  mb-xrpl.cfg  mb-xrpl.sqlite  secret.cfg

The logs are empty

I haven't reinstalled yet, but this should not be able to happen. If an autoupdate fails nothing should be deleted. All should be backed up first in the autoupdate before any deletion occurs and if it fails the previous files should be restored, automatically. 

Last heartbeat: 1702082802

"address": "rnpkkEEMDSYAg1G6eHWF66kKjrKoAqMbtV"

Some screenshots of what is left.

Secret has disapperaed
![image](https://github.com/EvernodeXRPL/evernode-host/assets/42262747/44ff094e-8304-48a6-9bc9-7445fdee773e)

Most files have vanished.

![image](https://github.com/EvernodeXRPL/evernode-host/assets/42262747/26588e9b-6f0b-4d76-93e6-e6d8bd6144d4)

it appears as if it was during the update that this failure occurred as the sa.cfg is titled version 0.7.2 and the dashboard showing me as still being on version 0.7.1

The timestamp of secret last saved is the most likely time and date that it occurred.

![image](https://github.com/EvernodeXRPL/evernode-host/assets/42262747/901bda9d-d7d9-458a-83e7-b33d89e57a10)

***Of note:

I have attempted to re-install and it acknowledges that an errored installation exists and asks to fix. It can't as there is no room. However, there is 56Gb of room.

Subsequent to this and much testing and research my entire system is reportiong no space yet it has more than adequate available space on a relatively new SMART passed NvmeSSD. 

This may very well be the issue in space but I am unsure why it is reporting none when much is available.

I have no log it appears also.

I have backed up my etc/sashimono folder and will reformat and reinstall with a prior backed up secret and .cfg file.

**Comment by kithminisg** [comment]
@rippleitinnz ,Can you check if you can find the auto-update logs with this command before formatting?
`journalctl -u evernode-auto-update | grep "Dec 09"`

**Comment by kithminisg** [comment]
@rippleitinnz, Could you also try writing some content to a dummy file inside the secret file location before addressing the space issue?

**Comment by rippleitinnz** [comment]
> journalctl -u evernode-auto-update | grep "Dec 09"

Only today's since reboot are available

![image](https://github.com/EvernodeXRPL/evernode-host/assets/42262747/72d1c6f2-de73-4607-a867-b15d233dc183)

**Comment by kithminisg** [comment]
So, what happens when you are trying to modify an existing file or write a new file in that secret location? 

Is this machine dedicated to Evernode, or does it contain some other software that will increase disk usage?

**Comment by kithminisg** [comment]
Your host's sa.cfg is at version 0.7.2, but the version in `/etc/sashimono/mb-xrpl/mb-xrpl.cfg` may still be 0.7.1 because the update was interrupted. This older version appears in the dashboard due to the incomplete update process.

**Comment by rippleitinnz** [comment]
That's clear, what I'm really concerned about, is at what point in the update process is my secret accessed? For my secret to go from a working secret to an empty file is of real concern. It also has a Xahaud node running on it which acts as a proxy, nothing else. 

It's clear that during the update process something went wrong and my evernode pretty much got deleted, along with my secret. 

I've replaced the disk (bigger) and am in the process of getting all up and running again and I'll put the disk into another machine to review it later.

**Comment by rippleitinnz** [comment]
@kithminisg - now that I've reinstalled evernode and it has a new account - how do I transfer across the previous secret.cfg and and mb-xrpl.cfg? 

Is it simply just swapping them out or how?

**Comment by kithminisg** [comment]
@rippleitinnz, you could have tried installation with your existing account after transferring the registration by executing the following command:
```
curl -fsSL https://stevernode.blob.core.windows.net/evernode-beta-v3/setup.sh | sudo bash -s transfer -i

```
Anyway, I think you need to follow the steps below in order to transfer your previous registration into this new host setup:

1. Backup your new account credentials (if required) and uninstall the existing installation.
2. Transfer your previous installation using the above command (Specify the transferee address as the same account. In your case, `rnpkkEEMDSYAg1G6eHWF66kKjrKoAqMbtV`).
3. [Reinstall Evernode](https://docs.evernode.org/en/latest/hosts/evernode-host.html#installation) using that transferee account details.

Swapping the account details is not a recommended approach, as it may lead to mismatches between your current installation and the previous one.

**Comment by rippleitinnz** [comment]
> @rippleitinnz, you could have tried installation with your existing account after transferring the registration by executing the following command:
> 
> ```
> 
> curl -fsSL https://stevernode.blob.core.windows.net/evernode-beta-v3/setup.sh | sudo bash -s transfer -i
> 
> 
> 
> ```
> 
> Anyway, I think you need to follow the steps below in order to transfer your previous registration into this new host setup:
> 
> 
> 
> 1. Backup your new account credentials (if required) and uninstall the existing installation.
> 
> 2. Transfer your previous installation using the above command (Specify the transferee address as the same account. In your case, `rnpkkEEMDSYAg1G6eHWF66kKjrKoAqMbtV`).
> 
> 3. [Reinstall Evernode](https://docs.evernode.org/en/latest/hosts/evernode-host.html#installation) using that transferee account details.
> 
> 
> 
> Swapping the account details is not a recommended approach, as it may lead to mismatches between your current installation and the previous one.
> 
> 
> 
> 

Thank you! I've followed this procedure now and have successfully reinstalled using my previous account and am back up and running

**Comment by mrnash88** [comment]
@rippleitinnz did you have to enter your secret to transfer your previous registration into the new host setup?

**Comment by rippleitinnz** [comment]
Yes I did. 

The process isn't that well documented but to understand it more fully, the transfer function is merely a deactivation of your existing account in a safe manner so that your old account and secret can be reused without conflict.

Make sure you keep a copy of your secret and account cfg then run this (they are both in the mb-XRPL directory)

Your account details are stored in /etc/sashimono/mb-xrpl/mb-xrpl.cfg and /etc/sashimono/mb-xrpl/secret.cfg.

curl -fsSL https://stevernode.blob.core.windows.net/evernode-beta-v3/setup.sh | sudo bash -s transfer -i

Once that is complete you'll be able to reinstall Evernode using your old account and secret.

**Comment by mrnash88** [comment]
Got it thank you. One thing I am unsure of is where do I find my registration NFT? I would appreciate it if you can tell me. Thanks in advance.

**Comment by BimsaraFernando** [comment]
Hi @rippleitinnz  ,
Thank you for your contribution. This issue was raised prior to the live release and has not had recent activity. We are closing it now. If this problem persists or if you have further questions, please open a new issue.
